### PR TITLE
doc: Kube + Helm installation

### PR DESCRIPTION
**Pull request by alram**
@liewegas @rootfs @dmick

### CODE REVIEW

**Comment by liewegas** [comment]
jenkins render docs

File: doc/start/kube-helm.rst [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+Create a ``~/ceph-overrides.yaml`` that will contain your Ceph configuration::
+
+    $ cat ~/ceph-overrides.yaml
+    network:
+      public:   172.21.0.0/20
+      cluster:   172.21.0.0/20
+
+    osd_devices:
+      - name: dev-sdd
+        device: /dev/sdd
```
Comment (vasukulkarni) [inline doc/start/kube-helm.rst]: probably a stupid question, what about hostname or IP? where are these thing created?

Comment (alram) [reply doc/start/kube-helm.rst]: Do you mean the nodes that will hold these devices? If so they are defined via label assignment

Comment (vasukulkarni) [reply doc/start/kube-helm.rst]: Thanks yes that is what i was looking for.

File: doc/start/kube-helm.rst [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+For each OSD node::
+
+    $ kubectl label node <nodename> ceph-osd=enabled ceph-osd-device-dev-sdb=enabled ceph-osd-device-dev-sdc=enabled
+
+Ceph Deployment
+===============
+
+Run the helm install command to deploy Ceph::
+
+    $ helm install --name=ceph local/ceph --namespace=ceph -f ~/ceph-overrides.yaml
```
Comment (vasukulkarni) [inline doc/start/kube-helm.rst]: Can we also specify how to use stable versions, if its coming from namespace?

Comment (dmick) [reply doc/start/kube-helm.rst]: namespace doesn't have anything to do with version; it's a kubernetes segregation mechanism.  install images are selected by 'images:' in values.yaml (and overridden at helm install time with -e or -f ceph-overrides.yaml)

Comment (vasukulkarni) [reply doc/start/kube-helm.rst]: Thanks @dmick

**Pull request merged**